crates/network-scanner-net/Cargo.toml

```diff
@@ -9,6 +9,15 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.79"
+crossbeam = { version = "0.8.4", features = ["crossbeam-channel"] }
+futures = "0.3.30"
+parking_lot = "0.12.1"
+polling = "3.3.2"
+
 socket2 = { version = "0.5.5", features = ["all"] }
-tokio = { version = "1.35.1", features = ["io-util","rt","sync"] }
+thiserror = "1.0.56"
 tracing = "0.1.40"
+tracing-subscriber = "0.3.18"
```

Comment:
issue: `tracing-subscriber` should not be a dependency of this crate. Maybe a dev-dependency?

Reply:
updated